runtime-infra/pn-ipc.yaml

```diff
@@ -848,3 +897,98 @@ Outputs:
     Value: !Ref LogsExporterRoleArn
   LogsExporterPolicyArn:
     Value: !Ref LogsExporterPolicyArn
+  AlbSecurityGroup:
+    Value: !Ref AlbSecurityGroup
+  CognitoUserPoolArn:
+    Value: !Ref CognitoUserPoolArn
+  CognitoClientId:
+    Value: !Ref CognitoClientId
+  ConfidentialInfoAccountId:
+    Value: !Ref ConfidentialInfoAccountId
+  HelpdeskAccountId:
+    Value: !Ref HelpdeskAccountId
+  OpenSearchArn:
+    Value: !Ref OpenSearchArn
+  OpenSearchEndpoint:
+    Value: !Ref OpenSearchEndpoint
+  LambdasBucketName:
+    Value: !Ref LambdasBucketName
+  LambdasBasePath:
+    Value: !Ref LambdasBasePath
+  EnvironmentType:
+    Value: !Ref EnvironmentType
+  Version:
+    Value: !Ref Version
+  SafeStorageAccountId:
+    Value: !Ref SafeStorageAccountId
+  SandboxSafeStorageBaseUrl:
+    Value: !Ref SandboxSafeStorageBaseUrl
+  ApiKeyTablesEncryptKeyId:
+    Value: !Ref ApiKeyTablesEncryptKeyId
+  VpcEgressSubnetsIds:
+    Value: !Ref VpcEgressSubnetsIds
+  ElasticacheEndpoint:
+    Value: !Ref ElasticacheEndpoint
+  ElasticacheSecurityGroup:
+    Value: !Ref ElasticacheSecurityGroup
+
+  CdnZoneId:
+    Value: !Ref CdnZoneId
+  PortalePaCertificateArn:
+    Value: !Ref PortalePaCertificateArn
+  PortalePfCertificateArn:
+    Value: !Ref PortalePfCertificateArn
+  PortalePfLoginCertificateArn:
+    Value: !Ref PortalePfLoginCertificateArn   
+  LandingCertificateArn:
+    Value: !Ref LandingCertificateArn 
+  PortalePgCertificateArn:
+    Value: !Ref PortalePgCertificateArn
+  PortaleStatusCertificateArn:
+    Value: !Ref PortaleStatusCertificateArn 
+  PortaleHelpdeskCertificateArn:
+    Value: !Ref PortaleHelpdeskCertificateArn
+  PortalePaDomain:
+    Value: !Ref PortalePaDomain
+  PortalePfDomain:
+    Value: !Ref PortalePfDomain
+  PortalePfLoginDomain:
+    Value: !Ref PortalePfLoginDomain    
+  LandingDomain:
+    Value: !Ref LandingDomain
+  PortalePgDomain:
+    Value: !Ref PortalePgDomain
+  PortaleStatusDomain:
+    Value: !Ref PortaleStatusDomain    
+  PortaleHelpdeskDomain:
+    Value: !Ref PortaleHelpdeskDomain
+  ReactAppUrlApi:
+    Value: !Ref ReactAppUrlApi
+  BoApiDnsName: 
+    Value: !Ref BoApiDnsName
+  HubLoginDomain:
+    Value: !Ref HubLoginDomain
+  TokenExchangeLambdaEnvironmentIssuer:
+    Value: !Ref TokenExchangeLambdaEnvironmentIssuer
+  TokenExchangeLambdaEnvironmentAllowedIssuer:
+    Value: !Ref TokenExchangeLambdaEnvironmentAllowedIssuer
+  TokenExchangeLambdaEnvironmentAllowedOrigin:
+    Value: !Ref TokenExchangeLambdaEnvironmentAllowedOrigin
+  TokenExchangeLambdaEnvironmentJwksMapping:
+    Value: !Ref TokenExchangeLambdaEnvironmentJwksMapping
+  TokenExchangeLambdaEnvironmentAcceptedAudience:
+    Value: !Ref TokenExchangeLambdaEnvironmentAcceptedAudience
+  TokenExchangeLambdaEnvironmentAudience:
+    Value: !Ref TokenExchangeLambdaEnvironmentAudience
+  ApikeyAuthorizerV2PDNDAudience:
+    Value: !Ref ApikeyAuthorizerV2PDNDAudience
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded API Gateway API Key. Hardcoded secrets can allow attackers to bypass authentication methods and perform malicious actions. Possible remediation approaches are:

	 1. Manage secret data using environment variables:
 		 a. Identify the secret data and store that data in environment variables on your local and production server. The setting process may vary depending on your OS and environment.
		 b. Replace hard-coded secrets with references to environment variables, e.g., `password = os.environ.get(&#39;PASSWORD&#39;)`.
	 2. Use AWS Secrets Manager to store, rotate, monitor, and control access to secrets. To retrieve secrets, you can replace hardcoded secrets in applications with a call to Secrets Manager APIs. To use Secrets Manager:
		 a. Visit [Secrets Manager](https://aws.amazon.com/secrets-manager/) on the AWS Management Console.
		 b. Choose the secret type on the console page and follow the instructions.
		 c. Use the code samples suggested in the console to retrieve the secret in your application.